debug_toolbar/panels/sql/tracking.py

```diff
@@ -60,10 +60,16 @@ def chunked_cursor(*args, **kwargs):
 
 def unwrap_cursor(connection):
     if hasattr(connection, "_djdt_cursor"):
+        if connection._djdt_cursor == connection.__class__.cursor:
+            del connection.cursor
+        else:
+            connection.cursor = connection._djdt_cursor
         del connection._djdt_cursor
+        if connection._djdt_chunked_cursor == connection.__class__.chunked_cursor:
+            del connection.chunked_cursor
+        else:
+            connection.chunked_cursor = connection._djdt_chunked_cursor
```

Comment:
How can we test that this is working as expected?

Additionally, adding summarizing the commit message would be helpful to future developers.

Reply:
Well, I had to implement it to get the multi-DB tests working :slightly_smiling_face:.  Not sure if it needs more testing than that...

Reply:
That's fair, thank you!